### PR TITLE
docs: fix refs to Python CDK manifest yaml

### PR DESCRIPTION
**Pull request by aaronsteers**
## What

Fix docs build failures:

- https://vercel.com/airbyte-growth/airbyte-docs/GfgmgDGHvcekLsKe4PcnSFd3jcox?filter=errors

Slack thread: https://airbytehq-team.slack.com/archives/C032Y32T065/p1732034232793509

Also disables CDK Sphinx builds. We expect this is being replaced with pdoc docs.

## Can this PR be safely reverted and rolled back?

- [ ] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Comment by aaronsteers** [comment]
Another error:

image

Working on it...

File: .readthedocs.yaml [hunk 1]
```diff
@@ -1,24 +0,0 @@
-# .readthedocs.yaml
```
Comment (natikgadzhi) [inline .readthedocs.yaml:1]: Thank you for cleaning this up!

File: docs/connector-development/tutorials/the-hard-way/build-a-connector-the-hard-way.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ demonstrate the following concepts in action:
 **This tutorial is meant for those interested in learning how the Airbyte Specification
 works in detail, not for creating production connectors**. If you're building a real source, you
 should start with using the [Connector Builder](../../connector-builder-ui/overview), or the
-[Connector Development Kit](https://github.com/airbytehq/airbyte/tree/master/airbyte-cdk/python/docs/tutorials).
+[Connector Development Kit](../../cdk-python/).
```
Comment (natikgadzhi) [inline docs/connector-development/tutorials/the-hard-way/build-a-connector-the-hard-way.md:23]: This I assume links to a docs page, not the repo? I think it's okay to link to the repo too if this breaks in weird ways.

**Review by natikgadzhi — APPROVED** [review]
Pre-approving to unblock, the direction is good here.

To follow up, please create an issue and assign it to me to actually request the yaml file from the repo at build time, so we don't have to sync the copy of the file here.

**Comment by natikgadzhi** [comment]
@aaronsteers for the second build error — it's fine to link to the neighboring repo I think.

**Comment by aaronsteers** [comment]
@natikgadzhi - Thanks for the review. I've confirmed that merging this resolved the build failures. I ended up using your suggestion of copying back the yaml file that was imported. We don't support https imports. There's a potential path to do so [here](https://webpack.js.org/configuration/experiments/#experimentsbuildhttp) but it's non-trivial and I didn't want to introduce new factors or potential security considerations in this fix.

I'll open a new issue as requested and assign to you.

**Pull request merged**